crates/nu-command/src/path/join.rs

```diff
@@ -90,6 +90,17 @@ the output of 'path parse' and 'path split' subcommands."#
                 example: r"'C:\Users\viking' | path join spams this_spam.txt",
                 result: Some(Value::test_string(r"C:\Users\viking\spams\this_spam.txt")),
             },
+            Example {
+                description: "Use relative paths, e.g. '..' will go up one directory",
+                example: r"'C:\Users\viking' | path join .. folder",
+                result: Some(Value::test_string(r"C:\Users\viking\..\folder")),
+            },
+            Example {
+                description:
+                    "Use absolute paths, e.g. '/' will bring you to the top level directory",
+                example: r"'C:\Users\viking' | path join / folder",
+                result: Some(Value::test_string(r"C:/folder")),
```

Comment:
This should result in `C:\folder` but when I tested it manually in the repl, it returned `C:/folder`. So, I think this is a bug, but I went ahead and added it with a forward slash since this PR isn't really about this bug.

Reply:
yeah exactly, let's leave this to another PR, when someone gets upset enough to fix `C:\folder` :relieved: :wink: